src/discordtipbot/discordbot.cr

```diff
@@ -440,6 +446,22 @@ class DiscordBot
     reply(msg, "Successfully turned #{memo} #{cmd[2]}") if @tip.update_config(memo, status, guild_id(msg))
   end
 
+  def check_config(msg : Discord::Message)
+    string = String.build do |str|
+      unless private?(msg)
+        mention = get_config(msg, "mention")
+        rain = get_config(msg, "rain")
+        soak = get_config(msg, "soak")
+        str << "Mentioning: #{mention}\nRaining: #{rain}\nSoaking: #{soak}\n"
+      end
+      str << "Minimum tip: #{@config.min_tip}\n"
+      str << "Minimum rain: #{@config.min_rain_total}\n"
+      str << "Minimum soak: #{@config.min_soak_total}"
```

Comment:
You can use `str.puts` and it will handle the newline at the end.

Reply:
I was looking for a method like that, but I somehow failed on finding it...